### DISCUSSION TITLE
Rclone

### DISCUSSION THREAD

**Discussion by null-character** [OnionUI/Onion#797] [Ideas]
Anyone else want rclone or some other way to cloud sync our saves?

The RG35XX can do it but that's Android. Looks.like rclone supports tons of architectures.

**Comment by schmurtzm** [comment]
It was already planned. In fact a beta is already in testing phase 😉
You’re right : rclone is available as is for arm : you don’t even need to build it to use it 😉

(I don’t mind that the rg35xx has it : garlic is not open source so it doesn’t help.)

**Reply by null-character** [reply]
Ok cool thanks for the info.
